Python/ResManager/pyResManager.cpp

```diff
@@ -28,11 +28,11 @@ PY_PLASMA_DEALLOC(ResManager)
 
 PY_PLASMA_INIT_DECL(ResManager)
 {
-    char* kwdlist[] = { "version", "preserveObjIDs", nullptr };
+    const char* kwdlist[] = { "version", "preserveObjIDs", nullptr };
```

Comment:
To match the pattern used elsewhere in PyHSPlasma (admittedly, it could probably be improved, but consistency is good too):
```suggestion
    char* kwdlist[] = { _pycs("version"), _pycs("preserveObjIDs"), nullptr };
```